### PR TITLE
Upgrade vagrant provision to actually work

### PR DESCRIPTION
**Pull request by turtleloveshoes**
r?

### CODE REVIEW

**Comment by mythmon** [comment]
This looks good. We walked through this together on Vidyo.

r+

File: bin/vagrant_provision.sh [hunk 1]
```diff
@@ -38,8 +38,9 @@ virtualenv $INSTALL_DIR/virtualenv
 chown -R vagrant $INSTALL_DIR/virtualenv
 source $INSTALL_DIR/virtualenv/bin/activate
 
+pip install -U 'pip<1.7'
```
Comment (safwanrahman) [inline bin/vagrant_provision.sh]: Why pip<1.7? @mythmon

**Comment by safwanrahman** [comment]
This does not install the front end Dependencies with bower!

**Comment by safwanrahman** [comment]
By the way, I had made a PR #2476 similar to this 4 months ago, but unfortunately it did not get merged!

File: bin/vagrant_provision.sh (hunk 1 above)
Comment (mythmon) [inline bin/vagrant_provision.sh]: Because pip 1.7 doesn't work with peep.

**Comment by mythmon** [comment]
Safwan, your PR 4 months was much larger in scope, and changed a lot more things. Additionally on one was able to get your patch working. Smaller patches are much easier to test and merge. Please review the comments in #2476, and see that this patch doens't have most of those problems.

File: bin/vagrant_provision.sh (hunk 1 above)
Comment (safwanrahman) [inline bin/vagrant_provision.sh]: This must be pip<7.0. Pip 7.0 does not work with peep and pip 1.7 is very old version.

**Comment by safwanrahman** [comment]
@mythmon, Though the patch was larger in scope, it was the thing that needed to be changed to get the vagrant work properly. Moreover fixing this patch dont let vagrant install in a 32 bit system.

> Additionally on one was able to get your patch working
>> Really? Can you confirm again? why no one commented there that its not working? I can clone the repo and I bet it will work.

@mythmon I have reviewed all the comments of the patch, and can you please check again why the PR was closed?  I have explained all the things why I have changed. And my change was quite similar with this moreover my patch had bower fixing which this patch does not fix.

File: bin/vagrant_provision.sh (hunk 1 above)
Comment (willkg) [inline bin/vagrant_provision.sh]: This PR has landed. If you'd like to make other changes, write up a new PR.

**Comment by willkg** [comment]
PR #2476 didn't work, had a lot of problems and I wasn't able to r+ it in the state it was in. For the reasons we listed in the comments, we closed it.

@mytmon did PR #2629 that fixes the bower bits which has also landed.

@safwan This PR has been merged and closed. If you'd like to make more changes, please submit a new PR.

**Pull request merged**